app/views/application_drafts/_student_fields.html.slim

```diff
@@ -5,7 +5,7 @@
   .form-inline
     = s.input :application_age, :collection => User::AGE, include_blank: true, label: 'Your age', disabled: !may_edit?(s.object), wrapper_html: { class: student_field_error_class(index, application_draft, :application_age)}, hint: student_field_error_hint(index, application_draft, :application_age)
   = s.input :application_gender_identification, label: 'Your gender self-identification', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_gender_identification) }, hint: (student_field_error_hint(index, application_draft, :application_gender_identification) ||  'No-one is excluded from applying on the basis of gender, but people who self-identify as female, or have experiences being socialized as female, are given preference during selection.')
-
+  = s.input :application_diversity, label: 'Tell us about any special circumstances which you want us to consider while reviewing your application', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_diversity) }, hint: (student_field_error_hint(index, application_draft, :application_diversity) || 'e.g. LGBTQI, people of color, people with disabilities, single parents, people taking care of family members etc.')
```

Comment:
@carpodaster I don't have a strong opinion on this. It seems to me that it's pretty hard to write more than 2000 chars about your special circumstances, but what do I know. The limit wouldn't harm anyone, I guess.